Enable system appropriate text AA, a la core LAFs

kschaefe opened the pull request:
Use reflection to set up and utilize the antialiasing features hidden in
the sun.\* packages.  The reflection is safe, ensuring that failures to 
initialize or utilize the reflected code do not leak.  The fallback for
reflection failures is the current painting approach.

This approach works with Oracle JREs.  Given the new Oracle JRE on Mac,
this code addresses Windows, Mac, and Linux releases effectively.  Note,
this change will not affect pre-Oracle, Apple JREs.

buildhive commented:
[Insubstantial » insubstantial #14](https://buildhive.cloudbees.com/job/Insubstantial/job/insubstantial/14/) FAILURE
Looks like there's a problem with this pull request
[(what's this?)](https://www.cloudbees.com/what-is-buildhive)

kschaefe committed:
Enable system appropriate text AA, a la core LAFs

Use reflection to set up and utilize the antialiasing features hidden in
the sun.* packages.  The reflection is safe, ensuring that failures to 
initialize or utilize the reflected code do not leak.  The fallback for
reflection failures is the current painting approach.

This approach works with Oracle JREs.  Given the new Oracle JRE on Mac,
this code addresses Windows, Mac, and Linux releases effectively.  Note,
this change will not affect pre-Oracle, Apple JREs.

buildhive commented:
[Insubstantial » insubstantial #15](https://buildhive.cloudbees.com/job/Insubstantial/job/insubstantial/15/) SUCCESS
This pull request looks good
[(what's this?)](https://www.cloudbees.com/what-is-buildhive)

kschaefe commented:
It was a bad merge of files on my end missing an import.  I have updated my HEAD, but can't figure out how to update this pull request.

shemnon commented:
Cloudbees is awesome.  For my next project I may force myself to do my own pull requests just to make cloudbees do pre-commit checks.

enwired commented:
This looks wonderful.  Thanks!

I had created my own fork of this project just to attempt to solve this problem.  But if this fix works, then I can stop trying.

@shemnon Do you think you could make a 7.3.1 release ?

shemnon commented:
I was going to take a stab at #99 before I started to spin out a 7.3.1.  This time around I'll do some code escrow on the snapshot.

kschaefe commented:
It looks as if this change was removed during the reopen/close of #100.  What's the reason for that?

shemnon commented:
It caused #103.  And the original cause of the bug was identified and fixed (it was a build problem).

kschaefe commented:
Ah.  I will check the new jar to ensure that it works on all of our platforms.

kschaefe commented:
I can confirm that the new jar works for us on Windows and Mac.  Thanks.

ahalma commented:
@kschaefe are the updated jars available for download?

shemnon commented:
only in snapshot.  There are a couple of other bugs I want to work through,
likely thanksgiving weekend.

On Fri, Nov 16, 2012 at 9:32 AM, ahalma notifications@github.com wrote:

> @kschaefe https://github.com/kschaefe are the updated jars available
> for download?
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/Insubstantial/insubstantial/pull/101#issuecomment-10452864.

## 

There is nothing that will hold me back.  I know who I am....
I remember wher I came from, and I feel stronger for knowing.
Zane, Ninja of Ice.  Ninjago S01E07

ahalma commented:
Looking forward to thanksgiving:)
